index.d.ts

```diff
@@ -61,6 +61,13 @@ declare namespace prettyMilliseconds {
 		@default false
 		*/
 		readonly formatSubMilliseconds?: boolean;
+
+		/**
+		Display time using colon notation: `5h 1m 45s`→ `5:01:45`.
+
```

Comment:
Would be useful to include some use-cases for this option. It's not always clear to the user when they would need/want this. (readme too).